### PR TITLE
[WI 150036] Update SDS_Views.md

### PR DESCRIPTION
**Pull request by GiselleLeeOSI**
Prepping the content for additional information (overriding auto-mapping and supported mapping)

### CODE REVIEW

File: Content_Portal/Documentation/SequentialDataStore/Reading_Data.md [hunk 1]
```diff
@@ -244,12 +244,12 @@ Data transformations are supported for all single stream reads, but transformati
 ### Reading with SdsStreamViews
 When transforming data with an SdsStreamView, the data read is converted to the *target type* specified in the SdsStreamView. Working with stream views is covered in detail in the [Stream Views](xref:sdsStreamViews) section.
 
-All stream view transformations are GET HTTP requests. The stream view is specified by appending the stream view identifier to requests to the transformation endpoint. For example the following request:
+All stream view transformations are HTTP GET requests. Specify the stream view ID at the end of the transformation endpoint in requests. For example, the following request returns the first event of the stream *transformed* to the target type (per stream view definition specified by `streamViewId`):
```
Comment (FreeJules) [inline Content_Portal/Documentation/SequentialDataStore/Reading_Data.md:247]: I would change this to - Specify the stream view ID at the end of the transformation endpoint in requests using `streamViewId` identifier.

File: Content_Portal/Documentation/SequentialDataStore/SDS_Views.md [hunk 2]
```diff
@@ -118,6 +115,31 @@ To update an SdsType of an SdsStream, define an SdsStreamView and do the followi
...
+| Numeric types              	| Yes           	| Yes                    	| No                	| No                         	| No                       	| No                                       	|
+| Nullable numeric types     	| Yes           	| Yes                    	| No                	| No                         	| No                       	| No                                       	|
+| Enumeration types          	| No            	| No                     	| Yes               	| Yes                        	| No                       	| No                                       	|
+| Nullable enumeration types 	| No            	| No                     	| Yes               	| Yes                        	| No                       	| No                                       	|
+| Object types               	| No            	| No                     	| No                	| No                         	| Yes*                     	| No                                       	|
+| Time-related types         	| No            	| No                     	| No                	| No                         	| No                       	| Yes**  	|
```
Comment (FreeJules) [inline Content_Portal/Documentation/SequentialDataStore/SDS_Views.md:138]: Remove Time related types, does not apply I double checked

File: Content_Portal/Documentation/SequentialDataStore/Reading_Data.md (hunk 1 above)
Comment (GiselleLeeOSI) [reply Content_Portal/Documentation/SequentialDataStore/Reading_Data.md]: Done. Please take a look and let me know if you approve.

File: Content_Portal/Documentation/SequentialDataStore/SDS_Views.md (hunk 2 above)
Comment (GiselleLeeOSI) [reply Content_Portal/Documentation/SequentialDataStore/SDS_Views.md]: Done. Please take a look and let me know if you approve.

**Review by FreeJules — APPROVED** [review]

**Pull request merged**